### DISCUSSION TITLE
Pictures used for dogs is outdated and is no longer up

### DISCUSSION THREAD

**Discussion by Jobrien0394** [community/community#6669] [Copilot Conversations]
I made a comment in my code for copilot to create an array of dogs with breed, age, and picture and the links that it pulled the pictures from wasn't up anymore and had a 404 error code.

**Comment by Xyphyn** [comment]
Because this depends on outside sources and isn't something that can be programmed locally, it may just have learned things from older repositories and put those inside your code.
